test/test_duckdb.py

```diff
@@ -0,0 +1,120 @@
+from sql_metadata import Parser
+
+
+def test_pivot_operator():
+    source = """
+SELECT *
+  FROM monthly_sales
+    PIVOT(SUM(amount) FOR MONTH IN ('JAN', 'FEB', 'MAR', 'APR'))
+      AS p
+  ORDER BY EMPID
+    """
+    parser = Parser(source)
+    assert ["monthly_sales"] == parser.tables
```

Comment:
this is the only test case that is affected by the changes I made in `token.py` and in fact it is not correcting regressed behavior due to the change in handling of the `pivot` keyword. I checked and it was parsing the same before those changes. Without the `last_keyword_pivot_operator` check this would also pull out `SUM` and `amount` as tables.